tests/unit/require.ts

```diff
@@ -1,12 +1,18 @@
 import * as assert from 'intern/chai!assert';
 import * as registerSuite from 'intern!object';
 
+interface LoaderError extends Error {
```

Comment:
This can be imported from `src/loader.ts`. Since it's just an interface, the compiler is smart enough to know that the import only needs to be done within the compiler and won't be added to the resulting JS.